packages/documentation-framework/components/themeSelector/themeSelector.js

```diff
@@ -104,7 +144,7 @@ export const ThemeSelector = ({ id }) => {
       shouldFocusToggleOnSelect
       onOpenChangeKeys={['Escape']}
     >
-      <SelectGroup>
+      <SelectGroup label={'Color scheme'} labelHeadingLevel="h2">
```

Comment:
Also depending if anything changes from above comment, but since we're giving a visible label here let's also update the aria-label of the SelectList to match this visible label.